src/pages/Lend/Filter/LendFilterPage.vue

```diff
@@ -38,13 +39,15 @@
 						:userToken="userId.toString()"
 						ref="aisConfigure"
 					/>
+					<algolia-search-box class="algolia-search-box-component" v-if="algoliaSearchEnabled" />
 					<selected-refinements
+						class="selected-refinements-component"
```

Comment:
These classes were added to reorder components for the largest breakpoint in the pinned filter experiment